src/components/Doc.tsx

```diff
@@ -18,14 +18,16 @@ type DocProps = {
     'sdk-support'?: {
       [key: string]: typeof headers
     }
+    docUrl?: string,
+    docUrlDescription?: string
```

Comment:
I think docUrlTitle or docUrlText might be a better name...? Description was confusion for me...

Reply:
Ahh, good call. Can make that better.

Or would a nested object be even clearer, since they are kindof a pair?

Reply:
I think it's OK as it is now.

Reply:
Sounds good. I went with `docUrlLinkText` for better clarity on how it's used.